tests/ModelParser/JMSParserTestCase.php

```diff
@@ -1061,7 +1061,7 @@ public function foo(): ?\DateTime
         $this->assertPropertyType(PropertyTypeDateTime::class, 'DateTime|null', true, $type);
         $this->assertSame('Y-m-d H:i:s', $type->getFormat(), 'Date time format should match');
         $this->assertSame('Europe/Zurich', $type->getZone(), 'Date time zone should match');
-        $this->assertSame('Y-m-d', $type->getDeserializeFormat(), 'Date time deserialize format should match');
+        $this->assertSame(['Y-m-d'], $type->getDeserializeFormats(), 'Date time deserialize format should match');
```

Comment:
we should keep the test for the deprecated method, and just add the test for the new method.